apps/design-system-docs/package.json

```diff
@@ -35,9 +37,14 @@
 		"@docusaurus/tsconfig": "^3.7.0",
 		"@docusaurus/types": "^3.7.0",
 		"@types/react": "^18.3.21",
+		"@wordpress/base-styles": "^5.23.0",
 		"@wordpress/browserslist-config": "^6.23.0",
+		"sass": "^1.54.0",
```

Comment:
What judgement did we use when providing `sass` as a dev dependency but `docusaurus-plugin-sass` as a regular dependency?

Reply:
Poor judgment 😇 Fixed in 3af9c4f